File: practices/string/print-pyramid.py
def pyramid(n):
    mid = (2 * n - 1) // 2
    for i in range(n):
        str = ""
        for j in range(2 * n - 1):
            if mid - i <= j <= mid + i:
                str += "*"
            else:
                str += " "
        print(str)

File: practices/string/print-pyramid.js
function pyramid(n) {
  const mid = Math.floor((2 * n - 1) / 2);
  for (let i = 0; i < n; i++) {
    let str = '';
    for (let j = 0; j < 2 * n - 1; j++) {
      if (mid - i <= j && mid + i >= j) {
        str += '#';
      } else {
        str += ' ';
      }
    }
    console.log(str);
  }
}